### DISCUSSION TITLE
Copilot Chat in PyCharm not accepting follow up questions

### DISCUSSION THREAD

**Discussion by BenTaub** [community/community#111144] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am entering questions into copilot chat (and I quite like it!) but I'm finding that when I receive an answer and then enter a follow up question, my question disappears and copilot does not respond. If I, however, start a new chat, copilot will respond to the question if I enter it again. Of course, I have to start a new chat to enter a follow up to that question. Etc.

Any suggestions on how to fix this?

**Comment by Murena7** [comment]
I have the same bug, it happened after the last plugin update.
